Web/scripts/fluid_properties.Consistency.py

```
@@ -1,42 +1,136 @@
+        regenerate_fragment_from_csv(fluid)
+    else:
+        print('fluid:', fluid, '- generating (backend=%s)' % backend)
+        csv_relpath = subdir + '/' + fluid + '-consistency.csv'
+        intro_rst = refprop_intro(fluid) if backend == 'REFPROP' else ''
+        file_string = template.format(backend=backend, fluid=fluid,
+                                      csv_relpath=csv_relpath, intro_rst=intro_rst, dpi=dpi)
+        file_path = os.path.join(plots_path, fluid + '.py')
+        with open(file_path, 'w') as fp:
+            fp.write(file_string)
+        try:
+            subprocess.check_call('python -u "' + fluid + '.py"', cwd=plots_path,
```

Ruff (0.15.14)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)